spec/components/onboarding_signal_link_spec.rb

```diff
@@ -8,7 +8,5 @@
   before(:each) { allow(Setting).to receive(:signal_server_phone_number).and_return('+4915112345678') }
 
   it { should have_css('.OnboardingSignalLink') }
-
-  # \u00a0 is a non-breaking space
-  it { should have_css('strong', text: "0151\u00a01234\u00a05678") }
+  it { should have_css('strong', text: '015112345678'.phony_formatted(normalize: :DE, spaces: ' ')) }
```

Comment:
This test was kinda of inexplicably failing as I didn't think adding a test runner for our system specs should affect our component specs. Having a closer look, the configuration I borrowed from [EvilMartians blog on System Tests](https://evilmartians.com/chronicles/system-of-a-test-setting-up-end-to-end-rails-testing), configures `:better_cuprite` as the `Capybara.default_driver`.  This causes slight differences as it's actually run in a headless browser.

I considered changing this and using `rack_test` for all tests which are not system tests, but the test suite ran in CI in 2min 45s, so I don't think speed is an argument for favoring rack_test.

Reply:
I actually really like the ability to run component tests in a real browser as well!